source/OptionsMenu.hx

```diff
@@ -58,6 +58,11 @@ class OptionsMenu extends MusicBeatState
 			new FlashingLightsOption("Toggle flashing lights that can cause epileptic seizures and strain."),
 			new WatermarkOption("Enable and disable all watermarks from the engine."),
 			new BotPlay("Showcase your charts and mods with autoplay.")
+		]),
+		
+		new OptionCategory("Saved Data", [
+			new LockWeeksOption("Lock all weeks. (exept Tutorial of course)")
```

Comment:
except (also make it more clear what this does so someone doesnt accidentally lock all weeks) (also maybe um;;; an option to unlock all? or make it a toggle? idk?)

Reply:
renamed it to `reset stroy progress`